openfecli/commands/quickrun.py

```diff
@@ -33,10 +33,9 @@ def _format_exception(exception) -> str:
 )
 @click.option(
     'output', '-o', default=None,
-    type=click.Path(dir_okay=False, file_okay=True, writable=True,
-                    path_type=pathlib.Path),
+    type=click.Path(dir_okay=False, file_okay=True, path_type=pathlib.Path),
     help="output file (JSON format) for the final results",
```

Comment:
Can you add extra info in the help docstring (or maybe there's a more appropriate place to do it?) to detail the behaviour we expect from this flag?

Reply:
I rewrote this to make it more precise - what additional info do you think is needed here?

Reply:
I was something like "file must not already exist" or whatever the standard text is for CLI tools - what you have is fine, i.e. now that it says "create"